app/views/users/show.html.erb

```diff
@@ -145,7 +145,7 @@
       </div>
     </div>
     <%= render "articles/user_metadata", context: "profile" %>
-    <div class="profile-dropdown">
+    <div class="profile-dropdown" data-info='{"id":<%= @user.id %>,"username":"<%= @user.username %>"}'>
```

Comment:
there's no need for this to be JSON, you're also not using the id, I wouldn't add it. a `data-username` with the username is fine :)